tools/hangar/__snapshots__/test_corpus/sdk_tests/api/delete.w_compile_tf-aws.md

```diff
@@ -84,7 +84,7 @@ module.exports = function({ $api_url, $http_DELETE, $http_Util }) {
   },
   "output": {
     "WING_TEST_RUNNER_FUNCTION_ARNS": {
-      "value": "[[\"root/Default/Default/test:http.delete and http.fetch can preform a call to an api\",\"${aws_lambda_function.testhttpdeleteandhttpfetchcanpreformacalltoanapi_Handler_98044CDF.arn}\"]]"
```

Comment:
Can you please explain why the IAM roles were deleted in this file after your change to `examples/tests/sdk_tests/api/delete.w`?

Reply:
ok, got it (thanks @tsuf239) - this is because the test doesn't run for this target